### PR TITLE
{Graph} Fix linter and style errors

### PR DESCRIPTION
**Pull request by jiasli**
**Related command**

Fix 

- Help messages
- Linter 
- Style
- CredScan

Corresponding doc update: https://github.com/MicrosoftDocs/azure-docs-cli/pull/3122

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/role/_help.py [hunk 1]
```diff
@@ -22,15 +22,18 @@
 short-summary: Create a web application, web API or native application
 long-summary: For more detailed documentation, see https://docs.microsoft.com/graph/api/resources/application
 examples:
-  - name: Create a native application with delegated permission of "access the AAD directory as the signed-in user"
+  - name: Create an application.
     text: |
-        az ad app create --display-name my-native --native-app --required-resource-accesses @manifest.json
+        az ad app create --display-name mytestapp
+  - name: Create an application that can fall back to public client with Microsoft Graph delegated permission Application.Read.All
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/role/_help.py:28]: Updated example from using outdated AD Graph permission to MS Graph permission.

File: src/azure-cli/azure/cli/command_modules/role/msgrpah/_graph_objects.py [hunk 2]
```diff
@@ -25,30 +28,44 @@
...
-    property_map is a map from argument name to property name, such as display_name -> displayName.
+
+_object_type_to_property_map = {
+    'application': _application_property_map,
+    'user': _user_property_map,
+    'group': _group_property_map
+}
+
+
+def set_object_properties(object_type, graph_object, **kwargs):
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/role/msgrpah/_graph_objects.py:59]: Extract public method `set_object_properties`.

**Comment by yonzhan** [comment]
Graph

File: src/azure-cli/azure/cli/command_modules/role/_help.py [hunk 3]
```diff
@@ -223,7 +226,7 @@
     For Application permissions, please use "ad app permission admin-consent"
 examples:
   - name: Grant a native application with permissions to access an existing API with TTL of 2 years
-    text: az ad app permission grant --id e042ec79-34cd-498f-9d9f-1234234 --api a0322f79-57df-498f-9d9f-12678
+    text: az ad app permission grant --id e042ec79-34cd-498f-9d9f-1234234 --api a0322f79-57df-498f-9d9f-12678 --scope Directory.Read.All
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/role/_help.py:229]: `--scope`'s default value `user_impersonation` is removed, as `user_impersonation` is very highly privileged.

File: src/azure-cli/azure/cli/command_modules/role/_help.py [hunk 4]
```diff
@@ -501,16 +499,16 @@
     short-summary: 'Number of years for which the credentials will be valid. Default: 1 year'
 examples:
   - name: Append a certificate to the service principal with the certificate string.
-    text: az ad sp credential reset --name myapp --cert "MIICoT..." --append
+    text: az ad sp credential reset --id myapp --cert "MIICoT..." --append
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/role/_help.py:502]: `az ad sp credential list` and `az ad sp credential delete` both use `--id`. Only `az ad sp credential reset` uses `--name`, so we unify them.

File: src/azure-cli/azure/cli/command_modules/role/_help.py [hunk 5]
```diff
@@ -573,7 +571,7 @@
 type: command
 short-summary: Create an Azure Active Directory user.
 parameters:
-  - name: --force-change-password-next-login
+  - name: --force-change-password-next-sign-in
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/role/_help.py:574]: This breaking change is undocumented in Graph doc (https://github.com/microsoftgraph/microsoft-graph-docs/issues/16819).

File: src/azure-cli/azure/cli/command_modules/role/custom.py [hunk 6]
```diff
@@ -1768,9 +1731,9 @@ def _delete_credential(graph_object, remove_password_func, patch_function, key_i
                 'keyCredentials': key_creds
             }
             patch_function(graph_object[ID], body)
-        print(graph_object)
-        raise CLIError("'{}' doesn't exist in graph object '{}'.".format(
-            key_id, graph_object[ID]))
+        else:
+            raise CLIError("No key credential found with keyId as '{}' in graph object '{}'.".format(
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/role/custom.py:1735]: Use the same error message as https://docs.microsoft.com/en-us/graph/api/application-removekey?view=graph-rest-1.0&tabs=http

File: src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py [hunk 7]
```diff
@@ -655,15 +655,15 @@ def test_user_scenario(self):
             'mail_nickname': 'graphusertest',
             'new_mail_nick_name': 'graphusertestupdate',
             'group': 'graphusertest_g',
-            'password': 'Test1234!!',
+            'password': self.create_random_name(prefix='password-', length=40),
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/role/tests/latest/test_graph.py:658]: Use random password and replace it in recordings to make it more secure.

**Review by evelyn-ys — APPROVED** [review]

**Review by calvinhzy — APPROVED** [review]

**Pull request merged**